appveyor.yml

```diff
@@ -74,7 +86,7 @@ test_script:
   - del /Q coverage.xml
 
   - ps: |
-      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache
+      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache --cov=storage
```

Comment:
I don't think this change is needed; the integration tests won't even import the `storage` module unless `CLCACHE_MEMCACHED` is set (and `py.test` correctly points out that no coverage data will be generated for `storage` since that file is not even used).